src/System.CommandLine/Parsing/ParseResult.cs

```diff
@@ -18,6 +18,15 @@ public class ParseResult
         private readonly IReadOnlyList<Token> _unparsedTokens;
         private readonly IReadOnlyList<Token> _unmatchedTokens;
 
+        public static readonly ParseResult Empty = new ParseResult(
```

Comment:
For now let's keep this internal.

Reply:
Since this won't be needed unless help is requested, it's better to make this a method and return the instance only when needed rather than allocating it up front all the time.